QUARKUS-2098.md

```diff
@@ -0,0 +1,27 @@
+# QUARKUS-2098 - Fix support of media types with suffixes
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-2098
+
+Test plan covers RESTEasy Reactive functionality provided by `quarkus-resteasy-reactive` and `quarkus-rest-client-reactive` extensions,
+with emphasis on multipart form data support.
+
+https://quarkus.io/version/main/guides/resteasy-reactive
+
+https://quarkus.io/version/main/guides/rest-client-reactive
+
+## Scope of the testing
+- Endpoints with mediatypes with suffix(e.g. application/test+json).
+
+### Impact on test suites and testing automation:
+Negligible. We will add several tests and corresponding endpoints.
+
+## Automated test development
+- Add test for mediatype with subtype and suffix
+- Add tests for mediatype with either subtype or suffix
+- Tests to make sure, that priority is
+  1. git subtype+suffix
```

Comment:
What does the 'git' mean here?

Reply:
Shows importance of not using IDEA's terminal for doing commits. Fixed.